### DISCUSSION TITLE
Date.getWeek()

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#3257] [Ideas]
Add getWeek method to Date Class that will return ISO8601 based week number.

Example of other programing languages: https://www.epochconverter.com/weeknumbers

**Comment by ajwfrost** [comment]
Sounds like a good idea ...
Copying the ISO-8601 description here:
> Week number according to the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) standard, weeks starting on Monday. The first week of the year is the week that contains that year's first Thursday (='First 4-day week').
